subsys/net/lib/openthread/platform/radio.c

```diff
@@ -384,7 +384,9 @@ void platformRadioProcess(otInstance *aInstance)
 		while ((tx_pkt = (struct net_pkt *)k_fifo_get(&tx_pkt_fifo,
 							      K_NO_WAIT))
 		      != NULL) {
-			openthread_handle_frame_to_send(aInstance, tx_pkt);
+			if (!IS_ENABLED(CONFIG_OPENTHREAD_RCP)) {
```

Comment:
I wonder if it's actually needed? The application is not supposed to send anything on its own in the case of NCP/RCP?

Reply:
Will test.

Reply:
Same problem here, this will pull in `openthread_handle_frame_to_send()` which has references to `otIp6NewMessage()` etc. which are not available. So either the call to `openthread_handle_frame_to_send()` is `if IS_ENABLED` checked or the interior of the function has that check.